geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -9990,6 +10011,43 @@ public EntryExpiryTask getEntryExpiryTask(Object key) {
     return br.getEntryExpiryTask(key);
   }
 
+  PartitionRegionConfig updatePRNodeInformation() {
+    if (cache.getDistributedSystem().getProperties().get(ENABLE_CLUSTER_CONFIGURATION) != "true") {
+      return null;
+    }
+    CacheLoader cacheLoader = basicGetLoader();
+    CacheWriter cacheWriter = basicGetWriter();
+    PartitionRegionConfig newConfig = null;
+    Region<String, PartitionRegionConfig> partitionedRegionRoot = getPRRoot();
+    if (partitionedRegionRoot != null) {
+      RegionLock rl = getRegionLock();
+      try {
+        rl.lock();
+        PartitionRegionConfig prConfig = partitionedRegionRoot.get(getRegionIdentifier());
+        if (prConfig != null) {
+          newConfig =
+              new PartitionRegionConfig(prConfig.getPRId(), prConfig.getFullPath(),
+                  prConfig.getPartitionAttrs(), prConfig.getScope(),
+                  prConfig.getEvictionAttributes(),
+                  this.getRegionIdleTimeout(), this.getRegionTimeToLive(),
+                  this.getEntryIdleTimeout(),
+                  this.getEntryTimeToLive(), prConfig.getGatewaySenderIds());
+
+          for (Node node : prConfig.getNodes()) {
+            if (node.getMemberId().equals(getMyId())) {
```

Comment:
My understanding is that every partition of the region will have this `if` check and the set the loader/writer in its node. Correct?

Reply:
Yes